include/megaapi_impl.h

```diff
@@ -2429,7 +2429,7 @@ class MegaApiImpl : public MegaApp
         static int typeComparator(Node *i, Node *j);
         static bool userComparatorDefaultASC (User *i, User *j);
 
-        char* escapeFsIncompatible(const char *filename);
+        char* escapeFsIncompatible(const char *filename, const char *dstPath);
         char* unescapeFsIncompatible(const char* name);
```

Comment:
I see these two are asymmetric now, we need to know the fileysystem for encoding but not decoding.   We could add some explanation for that in the megaapi.h comments, and also it makes me wonder what if the original filename included a % and had a couple of hex-compatible characters - we would not alter those for encode, but decode would collapse them to a single character.    Probably we should escape % itself in escapeFsIncompatible if is followed by two hex characters, so that unescape always gives you back the original?   thanks